apps/src/lib/kits/maker/BoardController.js

```diff
@@ -16,9 +16,23 @@ import {BOARD_EVENT_ALIASES} from './PlaygroundConstants';
 import TouchSensor from './TouchSensor';
 import Piezo from './Piezo';
 
+/**
+ * @typedef {Object} SerialPortInfo
+ * @property {string} comName (a port id) e.g. "COM3" or "/dev/ttyACM0"
+ * @property {string} manufacturer e.g. "Adafruit Circuit Playground"
+ * @property {string} productId e.g. "0x8011"
+ * @property {string} vendorId e.g. "0x239a"
+ */
+
 /** @const {string} */
 const CHROME_APP_ID = 'ncmmhcpckfejllekofcacodljhdhibkg';
 
+/** @const {string} Adafruit's vendor id as reported by Circuit Playground boards */
+export const ADAFRUIT_VID = '0x239a';
+
+/** @const {string} The Circuit Playground product id as reported by Circuit playground boards */
+export const CIRCUIT_PLAYGROUND_PID = '0x8011';
```

Comment:
how stable are these? If they were to change, do we have a good way of finding out that they did so?